blues/smartdart.py

```diff
@@ -66,28 +172,216 @@ def get_particle_masses(self, system, residueList=None):
         for index in range(len(residueList)):
             mass_list[index] = (np.sum(mass_list[index])).value_in_unit(unit.daltons)
         mass_list =  mass_list*unit.daltons
-        self.total_mass = total_mass
-        self.mass_list = mass_list
+        if set_self == True:
+            self.total_mass = total_mass
+            self.mass_list = mass_list
         return total_mass, mass_list
 
+    def dartsFromPDB(self, pdb_list, forcefield, residueList=None, basis_particles=None):
+        """
+        Used to setup darts from a pdb, using the basis_particles to define
+        a new coordinate system. ResidueList corresponds to indicies of
+        the ligand residues.
+
+        Arguments
+        ---------
+        pdb_list: list of str
```

Comment:
@sgill2 - this documentation will need cleaning up, as I can't tell how I would use it from the doc string. (What does it mean for the coordinates to be used as basis vectors? What is supposed to be in my PDB file? What will be done with it?)